pkg/cidata/cidata.TEMPLATE.d/boot/10-alpine-prep.sh

```diff
@@ -29,6 +29,8 @@ usermod -p '*' "${LIMA_CIDATA_USER}"
 
 # Alpine disables TCP forwarding, which is needed by the lima-guestagent
 sed -i 's/AllowTcpForwarding no/AllowTcpForwarding yes/g' /etc/ssh/sshd_config
+# Enable PAM so as to load /etc/environment via pam_env
+sed -i 's/#UsePAM no/UsePAM yes/g' /etc/ssh/sshd_config
```

Comment:
This looks unrelated to the PR (renaming the templates), but is fine because it is a tiny change and in a separate commit.

Reply:
I understand. It is just something we should have had even before; it was just CI that discovered it was missing. Anyways, I already wrote that I have no problem with including it in this PR. It is ready to be merged as soon as CI is green.